WordPress/src/main/java/org/wordpress/android/ui/media/MediaGridAdapter.java

```diff
@@ -269,11 +270,11 @@ private boolean inMultiSelect() {
     }
 
     private void loadNetworkImage(Cursor cursor, NetworkImageView imageView) {
-        String thumbnailURL = cursor.getString(cursor.getColumnIndex("thumbnailURL"));
+        String thumbnailURL = cursor.getString(cursor.getColumnIndex(WordPressDB.COLUMN_NAME_THUMBNAIL_URL));
 
         // Allow non-private wp.com and Jetpack blogs to use photon to get a higher res thumbnail
         if (mIsCurrentBlogPhotonCapable){
-            String imageURL = cursor.getString(cursor.getColumnIndex("fileURL"));
+            String imageURL = cursor.getString(cursor.getColumnIndex(WordPressDB.COLUMN_NAME_FILE_URL));
             if (imageURL != null) {
                 thumbnailURL = StringUtils.getPhotonUrl(imageURL, mGridItemWidth);
```

Comment:
PMFJI, but I just noticed the use of `StringUtils.getPhotonUrl()` here and I didn't realize that routine existed. We should remove that routine entirely and rely on [PhotonUtils.getPhotonImageUrl()](https://github.com/wordpress-mobile/WordPress-Utils-Android/blob/develop/WordPressUtils/src/main/java/org/wordpress/android/util/PhotonUtils.java#L47) since it's more versatile and can produce more bandwidth-friendly JPEGs.

In this case, you would use `PhotonUtils.getPhotonImageUrl(imageUrl, mGridItemWidth, 0)`.